libs/application/templates/european-health-insurance-card/src/lib/helpers/applicantHelper.ts

```diff
@@ -0,0 +1,238 @@
+import {
+  Application,
+  ExternalData,
+  FormValue,
+} from '@island.is/application/types'
+import { CardResponse, NationalRegistry, NridName } from '../types'
+
+function getObjectKey(obj: any, value: any) {
```

Comment:
Can i suggest using Record instead? What are the possible values? It looks like a string to me?